### PR TITLE
docs: Add API, Backend, and DB Naming Standards

### PR DESCRIPTION
**Pull request by christinejoymantua**
## What does this PR do?
- Introduces guidelines for naming conventions in APIs, backend services, and databases.

## Why did you choose this solution?
- Ensures consistency and clarity across the codebase.

## What is the dependency graph of the changes?
- README.md updated to include navigation to new standards document.
- New markdown document for specific naming standards created.

## How to test/verify?
- Review the updated README.md for navigation links.
- Check the content of the new standards documentation for clarity and completeness.

## Related ticket url:
- None provided.

### CODE REVIEW

**Comment by symphdevtools** [comment]
Hi! I'm generating PR title and description for you and will automatically apply it to your PR. I'll react :rocket: to this comment when done. Just sit back and chill!

When you have new changes to this PR and you want to regenerate the title and PR description, just comment `churva`!

Let me know if something is not working well, e.g.:
- Some parts of the title and description weren't preserved when generating title and description
- Disappearing screenshots or links
- Etc.

Thanks!

(RIP hatdog)

File: standards/apis_backend_db_naming_standards.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+```
+
+```
+CREATE TABLE user_roles (
+id INT AUTO_INCREMENT,
+user_id INT,
+role_id INT,
+PRIMARY KEY(id)
+);
+```
```
Comment (JHNLWHD) [inline standards/apis_backend_db_naming_standards.md:70]: Let's also add via `@Entities('table_names')` as example

**Review by JHNLWHD — CHANGES_REQUESTED** [review]

File: standards/apis_backend_db_naming_standards.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+class User {
+// ...
+}
+```
+
+## Database
+
+- **Database Tables** should be named in plural form and in snake_case (lowercase with underscores separating words). For example, a table for users would be 'users', and a table for user roles would be 'user_roles'.
+
+- **Database Columns** should be named in snake_case, which is all lowercase with underscores between words. For example, if you have a column for user email, it should be named 'user_email'.
```
Comment (johnreyev) [inline standards/apis_backend_db_naming_standards.md]:
```suggestion
- **Database Columns** should be named in snake_case, which is all lowercase with underscores between words. For example, if you have a column for user's email address, it should be named 'email_address'. 
```

File: standards/apis_backend_db_naming_standards.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+This document outlines the naming conventions used in our API routes, entities, and database tables.
+
+## API
+
+- **API Routes** should be named in plural form and in kebab-case (lowercase with hyphens separating words). For example, if you have a route for users, it should be named 'users'. If the route is for user roles, it should be named 'user-roles'.
+
+- **API Controllers** should be named in PascalCase with the word 'Controller' at the end. For example, if you have a controller for users, it should be named 'UsersController'. If the controller is for user roles, it should be named 'UserRolesController'.
+
+
+- **API Functions** should be named in camelCase. For example, if you have a function that gets user data, it should be named 'getUserData'. If the function is for updating user roles, it should be named 'updateUserRoles'.
```
Comment (johnreyev) [inline standards/apis_backend_db_naming_standards.md]:
```suggestion
- **API Functions** should be prefixed with verb and named in camelCase. For example, if you have a function that gets user data, it should be named 'getUserData'. If the function is for updating user roles, it should be named 'updateUserRoles'.
```

**Review by JHNLWHD — APPROVED** [review]
LGTM 👍🏼

**Pull request merged**